### PR TITLE
Throw error if handler is not defined in config

### PR DESCRIPTION
**Pull request by fourslashw**

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -88,6 +88,11 @@ function configureHandler(handler, options) {
 
 function getHandler(name, options) {
   var handler = options.handlers[name];
+  if (!handler) {
+    var errorStr = util.format('Handler "%s" is undefined.' +
```
Comment (seanmonstar) [inline lib/config.js]: I would say `'Handler "%s" is not defined in config"`

Comment (seanmonstar) [inline lib/config.js]: Also, seems there's a test error, since the `util` module is not in scope.

**Comment by fourslashw** [comment]
Added util require and changed error message.

**Comment by seanmonstar** [comment]
Cool, thanks!

**Pull request merged**